packages/core/src/rules/common/registry-dependencies.ts

```diff
@@ -9,16 +8,27 @@ export const RegistryDependencies: Oas3Decorator | Oas2Decorator = () => {
   return {
     DefinitionRoot: {
       leave() {
-        redoclyClient = new RedoclyClient();
+        if (!process.env.REDOCLY_DOMAIN) {
+          throw new Error('Redocly domain is not set');
```

Comment:
Just wondering if it will throw if I bundle something on my local machine, like: `npx @redocly/openapi bundle my-api.yaml` ?

Reply:
It shouldn't - the env var is setting in code.